### PR TITLE
fixes #10806 - removing content search

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html [hunk 1]
```diff
@@ -216,7 +216,7 @@ <h4 translate>Content Counts</h4>
         <tr ng-show="repository.content_type == 'yum'">
           <td translate>Packages</td>
           <td class="align-center">
-            <a ng-href="/katello/content_search#/!=&search[subgrid][repo_id]={{ repository.id }}&search[subgrid][type]=repo_packages">
+            <a ui-sref="products.details.repositories.manage-content.packages({productId: product.id, repositoryId: repository.id})">
```
Comment (jlsherrill) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html]: notice i changed this to go to the manage packages/puppet modules page and changed it to allow view_products, but hide the remove buttons if the user does not have permissions to edit_products

**Comment by jlsherrill** [comment]
Filed these issues as a result of this:
http://projects.theforeman.org/issues/10808
http://projects.theforeman.org/issues/10807

**Comment by jlsherrill** [comment]
This is ready for testing/merging since 2.3 was branched

**Comment by ehelms** [comment]
Tested locally for traces of weirdness, and the RPM seems to have built fine so ACK!

https://www.youtube.com/watch?v=eqyUAtzS_6M

**Pull request merged**